test/unit/cli/requireWebpackConfig.test.js

```diff
@@ -16,6 +16,11 @@ describe('requireWebpackConfig', () => {
     assert.deepEqual(requireWebpackConfig(configPath), expectedConfig);
   });
 
+  it('requires symlinked config file', () => {
+    const configPath = getConfigPath('.js', 'config-symlink');
```

Comment:
Test fails on windows due to the symlink.

Can we skip the test for windows with?
```js
if (process.platform === 'win32') {
  this.skip();
}


```